vocabularies/UI.xml

```diff
@@ -1071,8 +1115,22 @@ The trend is
           </Annotation>
         </Property>
         <Property Name="RequestAtLeast" Type="Collection(Edm.PropertyPath)" Nullable="false">
+          <Annotation Term="Common.PrimitivePropertyPath"/>
```

Comment:
```suggestion
```

Reply:
The "request at least" properties go into $select, which already excludes `@Analytics.AggregatableProperty#sum`. 

$select also allows complex properties, and it is not clear from the description of "request at least" that this should not be allowed.